packages/ember-toucan-form/src/components/toucan-form.hbs

```diff
@@ -18,6 +18,9 @@
         (ensure-safe-component this.CheckboxGroupComponent) form=form
       )
       Field=form.Field
+      FileInput=(component
+        (ensure-safe-component this.FileInputFieldComponent) form=form
```

Comment:
We should probably migrate our components to gts format (we only have this for tests I think), similar to https://github.com/CrowdStrike/ember-headless-form/pull/84, which (besides being nicer by itself) allows us to get rid of `ensure-safe-component`...

Reply:
I agree!